recipe/build.sh

```diff
@@ -1,14 +1,19 @@
 set -ex
 
-cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
-
-cargo build --release
-cargo install --path . --root "${PREFIX}"
-
+# cargo-c is a rust build tool that generates shared library and pkgconfig
+# it is installed into the build prefix
 # https://github.com/xiph/rav1e
 cargo install cargo-c
 
+# Required for cross-compiling
+export PKG_CONFIG_SYSROOT_DIR=$PREFIX
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
+
 # Hmm, does conda-forge have cargo-c??
 # https://github.com/Homebrew/homebrew-core/blob/7d7fc5432ee7b16e7a7ce9f85951052f7ad55e96/Formula/rav1e.rb
 cargo cinstall --library-type cdylib --release --prefix "${PREFIX}"
-cargo uninstall cargo-c
```

Comment:
don't you need this?

Reply:
The logs claim that cargo-c is installed into BUILD_PREFIX not PREFIX, so removing it is not necessary (at least on Windows checked UNIX).

Reply:
Yes. removing cargo-c is also unnecessary on unix because it is installed to build_prefix to it is not included in this conda-package.